django/db/backends/base/schema.py

```diff
@@ -1675,6 +1675,37 @@ def _deferrable_constraint_sql(self, deferrable):
         if deferrable == Deferrable.IMMEDIATE:
             return " DEFERRABLE INITIALLY IMMEDIATE"
 
+    def _unique_index_nulls_distinct_sql(self, nulls_distinct):
+        if nulls_distinct is False:
+            return " NULLS NOT DISTINCT"
+        elif nulls_distinct is True:
+            return " NULLS DISTINCT"
+        return ""
```

Comment:
This is just a preference thing but normally I'd put the positive case first then negative to aide readability. Is there a Django preference?

Some people also like to put the default in an else to align the returns.

eg:

```
if someting is true:
    ...
elif something is false:
    ...
else:
    ...
```

Reply:
I'd prefer `elif` → `if` on the second line 😁